backend/typescript/services/implementations/emailService.ts

```diff
@@ -20,31 +20,156 @@ class EmailService implements IEmailService {
     }
   }
 
-  async sendConfirmationEmail(to: string): Promise<void> {
+  async sendConfirmationEmail(
+    to: string,
+    registrantName: string,
+    campName: string,
+    sessionDates: string,
```

Comment:
question: is there a specific format that the session dates should be specified in? i.e. should the admins follow a date-month-year format or is it up to them how they want to input it?

Reply:
according to the email copy it should be [DD, MONTH YEAR - DD, MONTH YEAR] - so sessionDates will have to be passed in in that format

Reply:
ok yea makes sense. i guess it'll be enforced on the front end then?

Reply:
it'd be wherever these methods were being called - from the backend services